src/shared/components/layouts/styles/sidebar-layout.module.scss

```diff
@@ -35,6 +35,10 @@
     &::-webkit-scrollbar {
       display: none;
     }
+    // Hides the facets as they appear until the page content loads
+    @media only screen and (max-width: 640px) {
+      display: none;
+    }
```

Comment:
Similarly, I'd make it use the franklin mixin, although it's defined as min-width so you'd need to have it `display: none` by default, and in the media query it would be `display: initial` or something like this